keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -206,3 +211,37 @@ func IsCloudCore() (types.ModuleRunning, error) {
 
 	return types.NoneRunning, nil
 }
+
+// GetLatestVersion return the latest non-prerelease, non-draft version of kubeedge in releases
+func GetLatestVersion() (string, error) {
+	//Download the tar from repo
+	versionURL := "curl -k https://api.github.com/repos/kubeedge/kubeedge/releases/latest"
```

Comment:
define "https://api.github.com/repos/kubeedge/kubeedge/releases/latest" in const?

Reply:
done